typescript/sdk/test/gas/calculator.test.ts

```diff
@@ -206,7 +210,7 @@ describe('InterchainGasCalculator', () => {
         const validatorManager = Object.assign(
           {},
           // @ts-ignore - TODO more strongly type InterchainGasCalculator
-          contracts.inboxes[resolveDomain(originDomain)].validatorManager,
+          contracts.inboxes[resolveDomain(origin)].validatorManager,
```

Comment:
I think you can remove the `resolveDomain` here and L224 as well as the ts-ignores